src/game.cpp

```diff
@@ -13936,6 +13936,30 @@ overmap &game::get_cur_om() const
     return overmap_buffer.get( pos_om.x, pos_om.y );
 }
 
+std::string game::get_cur_om_id() const
```

Comment:
This name is a bit misleading. It's not the id of the current overmap (overmaps don't have ids), it's the id of the current overmap terrain.